ts/parts/ColumnSeries.ts

```diff
@@ -836,7 +836,7 @@ seriesType<Highcharts.ColumnSeries>(
                     // in visible range (#7046)
                     if (
                         point.y === threshold &&
-                        series.dataMax <= (threshold as any) &&
+                        (series.dataMax as any) <= (threshold as any) &&
```

Comment:
I doubt that this new `any` cast is necessary. If it is optional, just put a `series.dataMax &&` before, but it is actually not optional.

Reply:
Lazyness I guess. But we need to typecheck, as it can be 0.